AzFilesHybrid/AzFilesHybridUtilities.ps1

```diff
@@ -0,0 +1,30 @@
+function Write-FailedPSStyle(
```

Comment:
Nit: I would maybe recommend removing the "PSStyle" from these function names. From the outside, PSStyle is an implementation detail for the caller, no need to expose it in the name